packages/sdk/client-services/src/packlets/network/network-service.ts

```diff
@@ -2,17 +2,23 @@
 // Copyright 2022 DXOS.org
 //
 
+import { Stream } from '@dxos/codec-protobuf';
 import { NetworkManager } from '@dxos/network-manager';
-import { NetworkService, SetNetworkModeRequest } from '@dxos/protocols/proto/dxos/client/services';
+import { NetworkService, NetworkStatus, ChangeNetworkStatusRequest } from '@dxos/protocols/proto/dxos/client/services';
 
 export class NetworkServiceImpl implements NetworkService {
   constructor(private readonly networkManager: NetworkManager) {}
 
-  async getNetworkMode() {
-    return { mode: this.networkManager.networkMode };
+  subscribeNetworkStatus() {
```

Comment:
Is there a pattern for these methods?
e.g., createNetworkStatusSubscription?

Reply:
SubscribeTo<something> is a pattern I see in services. So I should add `To` part